v4/export/region_results.csv

```diff
@@ -0,0 +1,990 @@
+4,"","7480000000000000FF0500000000000000F8"
```

Comment:
what's this?

Reply:
It's region info read from TiDB. This is built by creating a table, inserting data and `select * from information_schema.tikv_region_status`. These three columns are region_id, start_key, end_key separately.

Reply:
oh it's test fixtures